src/Data/List/Properties.agda

```diff
@@ -365,6 +422,37 @@ module _ (f : A → B → C) where
   map-zipWith g (x ∷ xs) (y ∷ ys) =
     cong₂ _∷_ refl (map-zipWith g xs ys)
 
+  zipWith-flip : ∀ xs ys → zipWith f xs ys ≡ zipWith (flip f) ys xs
+  zipWith-flip []       []       = refl
+  zipWith-flip []       (x ∷ ys) = refl
+  zipWith-flip (x ∷ xs) []       = refl
+  zipWith-flip (x ∷ xs) (y ∷ ys) = cong (f x y ∷_) (zipWith-flip xs ys)
+
+module _ (f : A → A → B) where
+
+  zipWith-comm : (∀ x y → f x y ≡ f y x) →
```

Comment:
Should this be generalised to `(∀ x y → f x y ≡ g y x)`?

Reply:
that would be a breaking change (again, this was an existing proof that was moved to the bottom) but it can be done if you prefer

Reply:
Ah in which case, no it shouldn't be! Sorry thought it was a new proof!